cli/src/cli.rs

```diff
@@ -2331,11 +2341,11 @@ pub fn app<'ab, 'v>(name: &str, about: &'ab str, version: &'v str) -> App<'ab, '
                 .arg(
                     Arg::with_name("from")
                         .long("from")
-                        .value_name("FROM_PUBKEY")
```

Comment:
@garious , should this be `_ADDRESS`?

Reply:
Anything requiring a signature must be a pubkey

Reply:
This is on `create-address-with-seed` and does not require a signature

Reply:
It needs to be a pubkey though. It's the pubkey of the keypair passed to create-stake-account.